addons/fleet/security/ir.model.access.csv

```diff
@@ -20,3 +20,4 @@ fleet_vehicle_log_contract_access_right,fleet_vehicle_log_contract_access_right,
 fleet_service_type_access_right,fleet_service_type_access_right,model_fleet_service_type,fleet_group_manager,1,1,1,1
 access_fleet_vehicle_cost,access_fleet_vehicle_cost,model_fleet_vehicle_cost,fleet_group_manager,1,1,1,1
 access_mail_activity_type_fleet_manager,mail.activity.type.fleet.manager,mail.model_mail_activity_type,fleet.fleet_group_manager,1,1,1,1
+access_fleet_vehicle_assignation_log_fleet_group_user,fleet_vehicle_assignation_log fleet_group_user,fleet.model_fleet_vehicle_assignation_log,fleet.fleet_group_user,1,1,1,1
```

Comment:
Is it really intended that all employees can access and modify logs of all cars ?

Reply:
Zoveel vor me.